### ISSUE TITLE
Site is narrower in some pages

### ISSUE THREAD

**Issue by Sankra** [DIPSASA/dipsasa.github.io#40]
At least in http://tech.dips.no/about/

**Comment by ivarler** [comment]
Nope, it is not. Only looks that way because the scrollbar is gone in [http://tech.dips.no/about/](http://tech.dips.no/about/)
